code/controllers/subsystem/persistence.dm

```diff
@@ -223,3 +226,13 @@ SUBSYSTEM_DEF(persistence)
 		/obj/item/ammo_magazine/revolver/cmb = -1,
 		)
 
+/datum/season_datum/weapons/guns/smg_seasonal
+	name = "SMGs"
+	description = "Two different SMGs. A classic and a new guy."
+	item_list = list(
+		/obj/item/weapon/gun/smg/m25 = -1,
+		/obj/item/ammo_magazine/smg/m25 = -1,
+		/obj/item/weapon/gun/smg/mp7 = -1,
+		/obj/item/ammo_magazine/smg/mp7 = -1,
+		)
```

Comment:
why not put a uzi here

Reply:
uzi is already in cops and robbers as one of the choices so i didn't see the need to readd it to another smg list
i'd like the mp27 and mr25 tested a bit